app/controllers/ops_controller/settings/tags.rb

```diff
@@ -216,6 +216,7 @@ def ce_delete
     entry = @cat.entries.find(params[:id])
     audit = {:event => "classification_entry_delete", :message => "Category #{@cat.description} [#{entry.name}] record deleted", :target_id => entry.id, :target_class => "Classification", :userid => session[:userid]}
     if entry.destroy
+      MiqGroup.remove_all_managed_filters(entry.tag.name)
```

Comment:
@jrafanie I think it was available but as @imtayadeway suggested I moved it into Tag model